microcosm_elasticsearch/searching.py

```diff
@@ -166,19 +138,14 @@ def _query(self):
         Starts with the index's search function; customizes with the provided doc types, if any
 
         """
-        query = self._index_search()
-        query = query.filter("terms", **{self.doc_type_field: [type_name for type_name in self.doc_types]})
```

Comment:
We have use cases that don't want this default `terms` filter. So it can now be overwritten by making `_filter_terms` a noop.

The next step is to do something similar to `_auto_filters` from `microcosm-postgres` and make this feature opt-in.